packages/extender/.storybook/addons.js

```diff
@@ -0,0 +1,5 @@
+import '@storybook/addon-knobs/register';
```

Comment:
We doesn't have a global storybook for each projet in javascript component ? 🤔

Reply:
Nope, it was  only set in design-system-kit. But we could

Reply:
Because it's not the purpose of this story, I propose you to create a task about it :)

Reply:
https://jira.jahia.org/browse/BACKLOG-11788